README.md

```diff
@@ -727,6 +727,167 @@ Vue.component('courses-list', {
 
 :arrow_right: https://docs.angularjs.org/api/ng/service/$compile
 
+### Angular
+[Live Examples Here!](https://angular.io/generated/live-examples/lifecycle-hooks/eplnkr.html)
+
+#### [ngOnChanges()](https://angular.io/guide/lifecycle-hooks#onchanges)
+Respond when Angular (re)sets data-bound input properties. The method receives a `SimpleChanges` object of current and previous property values.
```

Comment:
Please add one free space:
- after and before headings (i.e. `#### [ngOnChanges()]`)
- after and before code snippets (` ``` `)